### DISCUSSION TITLE
Documentation snippets should be updated for latest version

### DISCUSSION THREAD

**Discussion by vontell** [LemonAppDev/konsist#659] [Report a Problem] [answered]
### Tell us what version of Konsist you use.

0.12.2

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

First off, love this library so far! Spending my day today integrating it into our codebase and trying things out.
This is a simple issue - I'm starting by just copying some of the snippets into a test class in a Spring project, and modifying them to see what sorts of tests I want to write. I noticed that some of these snippets use an outdated API. For example:

* `assertTrue` is now `assert`, and `assertFalse` is not `assertNot`

This is the main one, but I just wanted to mention that an audit of these snippets would be good to explore.

### Actual behaviour

The snippets from the docs do not work after copy-paste, and need slight updating.

### Expected behaviour

The docs have snippets that should work out of the box

**Comment by vontell** [comment]
Happy to provide more examples!

**Reply by igorwojda** [reply, marked as answer]
Thx for letting me know. 
A misstep on our side - we have prematurely released snippets to the upcoming version (actually `assertTrue` will be a new way to go soon). I have reverted this change.

**Reply by vontell** [reply]
No problem! Thanks @igorwojda, note while I was playing around with it, I also made a PR: https://github.com/LemonAppDev/konsist-documentation/pull/37 - feel free to close it if those have already been handled in another branch.

**Comment by igorwojda** [comment]
BTW if you find any interesting examples please share them with us on [Konsist Slack channel](https://kotlinlang.slack.com/archives/C05QG9FD6KS).

**Reply by vontell** [reply]
I certainly will! I'm just waiting to get access to the Slack, but I have quite a few examples I can share
